cmake/Modules/OpmFind.cmake

```diff
@@ -183,15 +183,18 @@ macro (find_and_append_package_to prefix name)
   # now a macro, and not a function anymore), so we must reinitialize
   string (TOUPPER "${name}" NAME)
   string (REPLACE "-" "_" NAME "${NAME}")
-
+  # Special treatment for MPI to deactivate CXX bindings 100%
+  string (REGEX REPLACE "^MPI" "MPI_C" NAME "${NAME}")
```

Comment:
this might cause MPI_MPI_C.

Reply:
True, if NAME would be MPI_C but it always is the package name converted to upper case.

Reply:
of course you are right, i somehow thought we were replacing flags in a list of dependencies. nm.